src/Atmos/Model/multiphysics_types.jl

```diff
@@ -252,10 +264,10 @@ function compute_rain_snow_params(atmos, state, aux, t, ts)
         state.ρ,
     )
     S_qi -= tmp1
-    S_ρe -= state.ρ * tmp1 * (I_i + Φ)
+    S_e -= tmp1 * (I_i + Φ)
     S_qr -= tmp2
-    S_ρe += state.ρ * _L_f * tmp2
-    S_qs += (tmp1 + tmp2)
+    S_e += tmp2 * _L_f
+    S_qs += tmp1 + tmp2
```

Comment:
Trusting that you got this right 😄

Reply:
yep. Adding all of this to qa and actually comparing results with a paper and not my common sense is next on the list...